apps/store/src/appComponents/StoreLayout.tsx

```diff
@@ -0,0 +1,57 @@
+import type { ReactNode } from 'react'
+import { Suspense } from 'react'
```

Comment:
Not a big deal but I thought we had lint rules to force import like this in a single line, no? 🤔 
 
```suggestion
import { Suspense, type ReactNode } from 'react'
```

Reply:
Sure, I forgot to change our eslint rules. Will do in separate PR